src/java.base/unix/classes/java/lang/ProcessEnvironment.java

```diff
@@ -86,11 +86,11 @@
         // to putenv/setenv from C will not be visible from Java code.
         byte[][] environ = environ();
 /*[IF CRIU_SUPPORT]*/
-        isCRIUEnabled = InternalCRIUSupport.isCRIUSupportEnabled();
+        isCRIUCapable = InternalCRIUSupport.isCRIUCapable();
```

Comment:
isnt `isCRIUCapable` always going to return true here?

Reply:
hmm, will have to rework on it, and verify via perf tests.
Marking this as draft for now.

Reply:
Updated [Invoke isCRIUSupportEnabledImpl() w/o loading j9criu native library](https://github.com/eclipse-openj9/openj9/pull/14917) which obsoletes this PR.

@tajila please have another look.